### PR TITLE
Add OpenVINO inference notebook and notebooks test

### PR DESCRIPTION
**Pull request by helena-intel**
Add a "playground" notebook which goes over the several options for doing inference with OpenVINO, and includes a short example for all currently supported tasks. 

Also added a test that runs weekly to verify that the notebooks still work. Apart from the weekly runs, the test only runs when notebooks are changed, not on every PR.

Try it out on Colab: https://colab.research.google.com/github/helena-intel/optimum-intel/blob/ov-inference-notebook/notebooks/openvino/optimum_openvino_inference.ipynb

### CODE REVIEW

**Review by echarlaix — APPROVED** [review]
Amazing work @helena-intel 🔥Just left a couple of minor comments

File: notebooks/openvino/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+install the requirements for running all notebooks, do `pip install -r
+requirements.txt`. If you do not want to install the requirements to run all the
+notebooks, you can also install the requirements for a specific notebook. They
+are listed at the top of each notebook file.
+
+The notebooks have been tested with Python 3.8 and 3.10 on Ubuntu Linux.
+
+| Notebook     |      Description      |   |   |
+|:----------|:-------------|:-------------|------:|
+| [How to do inference with the OpenVINO integration](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb) | Explains how to do inference with OpenVINO and includes brief examples for all supported tasks| [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb)| [![Open in AWS Studio](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb)|
```
Comment (echarlaix) [inline notebooks/openvino/README.md:13]:
```suggestion
| [How to run inference with the OpenVINO](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb) | Explains how to export your model to OpenVINO and how to easily run inference with OpenVINO Runtime| [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb)| [![Open in AWS Studio](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/huggingface/optimum-intel/blob/main/notebooks/openvino/optimum_openvino_inference.ipynb)|
```

File: notebooks/openvino/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# 🤗 Optimum OpenVINO Notebooks
```
[the remark this answers is not in the document]
Comment (helena-intel) [reply notebooks/openvino/README.md:1]: I have seen users be confused about neural compressor in the README and I think it's nice to have a place to point people to with notebooks about OpenVINO. If we want to have one place with the links, I would prefer a separate README for OpenVINO and INC. 
But we can also discuss this later if you prefer, I can remove this README from the PR for now.

[the remark this answers is not in the document]
Comment (echarlaix) [reply notebooks/openvino/README.md:1]: I see your point, we can either split it into two tables in `notebooks/README.md` or have both tables separated in `notebooks/openvino/README.md`  and `notebooks/neural_compressor/README.md`, I'm up for both suggestions ! Open to do it this PR or in an upcoming one

**Review by AlexKoff88 — APPROVED** [review]

File: notebooks/openvino/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (echarlaix) [reply notebooks/openvino/README.md:1]: what option do you prefer @helena-intel ?

[the remark this answers is not in the document]
Comment (helena-intel) [reply notebooks/openvino/README.md:1]: My preference is a separate OpenVINO and INC README. Happy to change it in this PR or a new one if you agree. Should I then remove the `notebooks/README.md` file or do you prefer a 'stub' README with links to the INC and OpenVINO READMEs?

**Pull request merged**